<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentSliceGeometry</class>
 <widget class="qMRMLWidget" name="SegmentSliceGeometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>372</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="parameterSetLabel">
     <property name="text">
      <string>Parameter set: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="parameterNodeSelector" native="true">
     <property name="toolTip">
      <string>Pick node to store parameter set</string>
     </property>
     <property name="nodeTypes" stdset="0">
      <stringlist>
       <string>vtkMRMLScriptedModuleNode</string>
      </stringlist>
     </property>
     <property name="showHidden" stdset="0">
      <bool>true</bool>
     </property>
     <property name="baseName" stdset="0">
      <string>SegmentSliceGeometry</string>
     </property>
     <property name="renameEnabled" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="basicCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Segmentation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="segmentationSelector" native="true">
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="editEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Volume axis:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="axisSelectorBox">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>row</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>column</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>slice</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="ctkCollapsibleButton" name="basicCollapsibleButton_2" native="true">
     <property name="text" stdset="0">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Results table:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="tableSelector" native="true">
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLTableNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>false</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="editEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="renameEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="noneDisplay" stdset="0">
         <string>Create new table</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SegmentSliceGeometry</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>parameterNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SegmentSliceGeometry</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>tableSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SegmentSliceGeometry</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>segmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
